generator/internal/mixin_utils_test.cc

```diff
@@ -26,8 +26,41 @@ using ::google::protobuf::FileDescriptor;
 using ::google::protobuf::ServiceDescriptor;
 using ::testing::AllOf;
 using ::testing::Contains;
+using ::testing::ElementsAre;
+using ::testing::Eq;
+using ::testing::Field;
 using ::testing::NotNull;
 
+bool is_http_rule_expected(google::api::HttpRule const& http_rule,
+                           std::string const& body, std::string const& path,
+                           google::api::HttpRule::PatternCase pattern_case) {
+  if (http_rule.body() != body) return false;
+  if (http_rule.pattern_case() != pattern_case) return false;
+  if (pattern_case == google::api::HttpRule::kGet) {
+    return http_rule.get() == path;
+  }
+  if (pattern_case == google::api::HttpRule::kPost) {
+    return http_rule.post() == path;
+  }
```

Comment:
This assumes we only ever test gets/posts, which might be true, but seems not ideal.

Reply:
Replaced by IsProtoEqual.